travis_ci_flake8.cfg

```
@@ -0,0 +1,135 @@
+[flake8]
+# temporarily disable violations for existing code until they are cleaned up
+ignore =
```

This should be

```suggestion
extend-ignore
```

to include the default ignores. (see https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-extend-ignore)

fixed in https://github.com/senaite/senaite.core/pull/2085